flag.go

```diff
@@ -194,10 +199,12 @@ func sortFlags(flags map[NormalizedName]*Flag) []*Flag {
 // "--getUrl" which may also be translated to "geturl" and everything will work.
 func (f *FlagSet) SetNormalizeFunc(n func(f *FlagSet, name string) NormalizedName) {
 	f.normalizeNameFunc = n
+	f.order = f.order[:0]
```

Comment:
When you do `f.order = []NormalizedName{}` you create one more array and make one more allocation. But if you do `f.order = f.order[:0]` you reuse the current array but just delete all elements in it.